src/renderers/webgl/WebGLCapabilities.js

```diff
@@ -1,7 +1,12 @@
+import { FloatType, HalfFloatType, RGBAFormat, UnsignedByteType } from '../../constants.js';
+import { WebGLUtils } from './WebGLUtils.js';
+
 function WebGLCapabilities( gl, extensions, parameters ) {
 
 	let maxAnisotropy;
 
+	const utils = new WebGLUtils( gl, extensions );
```

Comment:
Please do not create an instance of `WebGLUtils` here. Instead, pass the one from `WebGLRenderer` to the ctor of `WebGLCapabilities`.

Reply:
fixed